docs/apache-airflow/tutorial.rst

```diff
@@ -376,3 +376,133 @@ Here's a few things you might want to do next:
       - Review the :ref:`List of operators <pythonapi:operators>`
       - Review the :ref:`Macros reference<macros>`
     - Write your first pipeline!
+
+
+Lets look at another example; we need to get some data from a file which is hosted online and need to insert into our local database. We also need to look at removing duplicate rows while inserting.
+
+Initial setup
+'''''''''''''
+We need to have docker and postgres installed.
+We will be using this `docker file <https://airflow.apache.org/docs/apache-airflow/stable/start/docker.html#docker-compose-yaml>`_
+Follow the instructions properly to set up Airflow.
+
+Create a Employee table in postgres using this
+
+.. code-block:: sql
+
+  create table "Employees"
+  (
+      "Serial Number" numeric not null
+   constraint employees\_pk
+              primary key,
+      "Company Name" text,
+      "Employee Markme" text,
+      "Description" text,
+      "Leave" integer
+  );
+
+  create table "Employees_temp"
+  (
+      "Serial Number" numeric not null
+   constraint employees\_pk
+              primary key,
+      "Company Name" text,
+      "Employee Markme" text,
+      "Description" text,
+      "Leave" integer
+  );
+
+I have broken this down into 3 steps get data, insert data, merge data.Lets look at the code:
+
+.. code-block:: python
+
+  def get_data():
+      url = "https://docs.google.com/uc?export=download&id=1a0RGUW2oYxyhIQYuezG_u8cxgUaAQtZw"
+
+      payload = {}
+      headers = {}
+
+      response = requests.request("GET", url, headers=headers, data=payload)
+
+      with open('/usr/local/airflow/dags/files/employees.csv', 'w') as file:
+          for row in response.text.split('\n'):
+              file.write(row)
```

Comment:
```suggestion
      response = requests.request("GET", url)

      with open('/usr/local/airflow/dags/files/employees.csv', 'w') as file:
          for row in response.text.split('\n'):
              file.write(row)
```

Since we are not passing them